workshop/content/lab1.4_developer.md

```diff
@@ -7,17 +7,19 @@ The second major win is that deploying to a development kubernetes environment b
 
 ## Deploy Service
 
-1.  Create project/namespace
+1.  Create project/namespace with assigned user number
 ```
-oc new-project hello
+# replace X with your assigned number
+export USER_NUMBER=X
+oc new-project user-$USER_NUMBER
 ```
 
 2.  Build the project
 ```
 oc new-build python:3.6~https://github.com/RedHatGov/serverless-workshop-code --name hello-python --context-dir=hello-python
```

Comment:
This resulted in an error for me:

```error: the image match "python:3.6" for source repository "https://github.com/RedHatGov/serverless-workshop-code" does not appear to be a source-to-image builder.```

Adding the `--strategy=docker` worked:

```oc new-build python:3.6~https://github.com/RedHatGov/serverless-workshop-code --name hello-python --context-dir=hello-python --strategy=docker```

Reply:
Running through it now, I'm getting the same error.  Updating instructions.

Reply:
👍